docs/creating-your-site.md

```diff
@@ -14,13 +14,14 @@ mkdocs new .
 
 Alternatively, if you're running Material for MkDocs from within Docker, use:
 
-=== "Unix, Powershell"
+=== "Unix"
 
     ```
     docker run --rm -it -v ${PWD}:/docs squidfunk/mkdocs-material new .
+    sudo chown -R $UID:$UID ./*
     ```
 
-=== "Windows"
+=== "Windows, Powershell"
```

Comment:
I'm not a Windows user, but as I understand there's WIndows Command Prompt and Windows Powershell, the latter of which has the same behavior as Unix (macOS, Linux)

Reply:
Well I have to say: Don't know.
But if I'm not wrong, there is no such thing as "Powershell" at Unix.

Reply:
No there isn't, that's correct. It might be a little misleading actually, but I didn't want to copy the same command 4 times for: Linux, macOS, Windows, Windows Powershell. That's overkill. The first tab reads as "Unix OR Powershell"

Reply:
Ahhh, I got it. Well, at least I got it wrong.
Maybe change to "Unix/Powershell | Windows". Or maybe reduce to Unix command.